ios/inputParser/InputParser.mm

```diff
@@ -603,6 +629,14 @@ - (NSArray *)getTextAndStylesFromHtml:(NSString *)fixedHtml {
       [styleArr addObject:@([BoldStyle getStyleType])];
     } else if([tagName isEqualToString:@"i"]) {
       [styleArr addObject:@([ItalicStyle getStyleType])];
+    } else if([tagName isEqualToString:@"img"]) {
+      [styleArr addObject:@([ImageStyle getStyleType])];
+      // cut only the uri from the src="..." string
+      NSString *uri = [params substringWithRange:NSMakeRange(5, params.length - 6)];
```

Comment:
I think we could do the same trick as with the links; use a regex for finding `src` and if there is none (or empty `src`), then we don't recognize the tag. This way we can dodge any crashes from not getting the params for the tag.